hyperapp.d.ts

```diff
@@ -54,42 +59,48 @@ export type ActionResult<State extends Hyperapp.State> =
   | null
   | void
 
-export type ActionCallers<
+export interface Action<State extends Hyperapp.State> {
+  (data: any): ActionResult<State>
```

Comment:
Can we make this generic? I feel like the typing for `data` here is too loose.

Reply:
Done, but this is not super useful since `Action<>` is only used in `Actions<>` and I have to set the type to `any` there (it may be something else than an object).